hoard/client.py

```diff
@@ -53,5 +55,19 @@ class DSpaceClient:
 
 
 class OAIClient:
-    def __init__(self, url):
-        self.url = url
+    def __init__(self, source_url: str, format: str, set: str = None) -> None:
```

Comment:
Since you added the `format` parameter here, you will also need to add it in `cli.py` where this is called or mypy will keep giving you errors.